option.go

```diff
@@ -149,8 +149,49 @@ func panicsIfNotCorrectType(openapiParam *openapi3.Parameter, exampleValue any)
 	return exampleValue
 }
 
-// Registers a parameter for the route. Prefer using the [Query], [QueryInt], [Header], [Cookie] shortcuts.
-func OptionParam(name string, options ...func(*OpenAPIParam)) func(*BaseRoute) {
+// Declare a response header for the route.
+// This will be added to the OpenAPI spec, under the 200 response.
```

Comment:
This a bit confused here. Is this only for 200 responses. Is this sentence only suppose to be on context of the example below?

Reply:
Yes I'll update it

Reply:
Updated